common/ledger/blockledger/fileledger/impl.go

```diff
@@ -110,6 +110,16 @@ func (fl *FileLedger) Height() uint64 {
 	return info.Height
 }
 
+// GetCurrentBlockHash returns the hash of the current block's header
```

Comment:
why do we need to expand the API of the ledger? You can just retrieve the current block, now it's cached.

Reply:
In the orderer we can, but in the peer we sometime can't, because of a snapshot. Hence, the BFTDeliverer eexpects this API, as it is commot to the peer as well.

Reply:
Can you explain where in the production code this is used? I can't find it in the PR.

Reply:
After a second look, you are correct, this is not needed in the orderer. 
I removed the respective changes to the `fileledger`.